profile/profile.yaml

```diff
@@ -0,0 +1,85 @@
+# Preferred name (Characters only)
+name: "Preet Khasakia"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_PreetKhasakia/
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: I am a recent graduate out of Ontario Tech University in Software Engineering. Throughout my years in undergrad, I was exposed to many different assignments and projects that allowed to gain knowledge in various aspects of programming essentials. Coursework in distributed systems, cloud computing and Capstone I/II had a great deal of hands-on experience with the ability to grow in a team-oriented work environment. I have recently joined Jarvis as a Data Engineer trainee to improve on my skillset and familiarize myself with real-world projects to continue to grow and apply knowledge throughout the program. 
+
+# Group your skills by level.
+# Your skills can be anything software related (e.g. language, frameworks, methodologies, software, etc.).
+skills:
+  # At least 5 skills
+  proficient:
+    - Java
+    - Linux/Bash
+    - RDBMS/SQL
+    - Agile/Scrum
+    - Git
+  # At least 5 skills
+  competent:
+    - Python
+    - C/C++
+    - PSQL
+    - HTML
+    - CSS
+  # At least 5 skills
+  familiar:
+    - Docker
+    - Bash
+    - Arduino
+    - AWS
+    - Heroku
+ 
+# List of Jarvis projects
+jarvis_projects:
+  - name: "Cluster Monitor"
+    git_url: "/linux_sql"
+    
+    description: "Designed and implemented a linux cluster agent to preserve and monitor real-time data of nodes running on CentOS. Application performs real-time data collection of hardware specifications and CPU memory usage. Created a Data Definition Language (DDL) script to hold specified data into tables used in the database. Modified 'host_info.sh' and 'host_usage.sh' to retrieve the necessary and insert it to the database."
+
+  - name: "Core Java Apps"
+    git_url: "/core_java"
+    description: 
+        - "Grep App: Implemented a java application that mimics the Linux 'grep' command to search and retrieve matching strings from files. Technologies used for this project are Maven, Java, Docker as a deployment tool and IntelliJ IDE. Implementation of backend code uses Encapsulation along with an interface to provide a flexible and organized implementation of the application. Deployment is done by dockerizing the app which includes building an image locally, running the docker container and push the image onto Docker Hub for verification of the image." 
```

Comment:
- "Grep App: Implemented a java application that mimics the Linux 'grep' command to search and retrieve matching strings from files. Technologies used for this project are Maven, Java, Docker as a deployment tool and IntelliJ IDE. Implementation of backend code uses Encapsulation and an interface to provide a flexible and organized application implementation. Deployment was done by dockerizing the app, which includes building an image locally, running the docker container, and pushing the image onto Docker Hub to verify the image." 

fixed grammar